pandas/core/arrays/integer.py

```diff
@@ -144,6 +144,13 @@ class IntegerArray(NumericArray):
 -------
 None
 
+See Also
+--------
+Int8Dtype : For np.int8.
```

Comment:
Could you say `n-bit nullable integer type` instead?

Reply:
yeah. let me do that.